### ISSUE TITLE
character encoding problem using utf-8 characters in log messages

### ISSUE THREAD

**Issue by pvojnisek** [AppDaemon/appdaemon#1375]
Using utf-8 characters in the python code:
`
self.log('éáűőúöüóí')
`
Causes incorrectly encoded characters in Supervisor/AppDaemon log.

**Comment by zerg960** [comment]
yeah, I can't output log messages written in Japanese. Everything comes out as "������������"

**Comment by zerg960** [comment]
Use `ascii_encode = False`:
```
    adapi.log(f'{message.author}: {message.content}', ascii_encode = False)
```

I don't know why that's not the default behavior.

**Comment by runningman84** [comment]
@acockburn can you include a fix in the upcoming release?

**Comment by runningman84** [comment]
Yes the workaround works…

**Comment by acockburn** [comment]
Thanks for the confirmation, I'll switch the default then.

**Comment by jsl12** [comment]
This is somewhat [documented](https://appdaemon.readthedocs.io/en/latest/AD_API_REFERENCE.html#appdaemon.adapi.ADAPI.log) already, but I just added a config option to control the default of whether `ascii_encode` gets set or not.

You would do this in your `appdaemon.yaml` to make it `False` by default, which would save you from having to pass that argument in every `self.log` call
```yaml
appdaemon:
  ... # rest of config
  ascii_encode: False
```
